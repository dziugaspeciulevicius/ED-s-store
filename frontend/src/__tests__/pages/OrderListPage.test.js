import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import OrderListPage from "../../pages/OrderListPage.component"
jest.mock("../../pages/OrderListPage.component");

describe("<OrderListPage />", () => {
    it("Renders <OrderListPage /> component correctly", () => {
      // shallow creates instance of a component
      expect(shallow(<OrderListPage />)).toMatchSnapshot();
    });
});

