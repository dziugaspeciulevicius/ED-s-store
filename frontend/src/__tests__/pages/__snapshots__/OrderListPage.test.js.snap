// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderListPage /> Renders <OrderListPage /> component correctly 1`] = `ShallowWrapper {}`;
